util/Migrator/DbScripts/2025-06-01_00_AddSignatureKeyPairTable.sql

```diff
@@ -0,0 +1,320 @@
+CREATE TABLE [dbo].[UserSignatureKeyPair] (
+    [Id]                        UNIQUEIDENTIFIER NOT NULL,
+    [UserId]                    UNIQUEIDENTIFIER NOT NULL,
+    [SignatureKeyPairAlgorithm] TINYINT NOT NULL,
+    [SigningKey]                VARCHAR(MAX) NOT NULL,
+    [VerifyingKey]              VARCHAR(MAX) NOT NULL,
+    [CreationDate]              DATETIME2 (7) NOT NULL,
+    [RevisionDate]              DATETIME2 (7) NOT NULL,
+    CONSTRAINT [PK_UserSignatureKeyPair] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_UserSignatureKeyPair_User] FOREIGN KEY ([UserId]) REFERENCES [dbo].[User] ([Id])
+);
+GO
```

Comment:
```suggestion
IF OBJECT_ID('[dbo].[UserSignatureKeyPair]') IS NULL
BEGIN
    CREATE TABLE [dbo].[UserSignatureKeyPair]
    (
        [Id] UNIQUEIDENTIFIER NOT NULL,
        [UserId] UNIQUEIDENTIFIER NOT NULL,
        [SignatureKeyPairAlgorithm] TINYINT NOT NULL,
        [SigningKey] VARCHAR(MAX) NOT NULL,
        [VerifyingKey] VARCHAR(MAX) NOT NULL,
        [CreationDate] DATETIME2 (7) NOT NULL,
        [RevisionDate] DATETIME2 (7) NOT NULL,
        CONSTRAINT [PK_UserSignatureKeyPair] PRIMARY KEY CLUSTERED ([Id] ASC),
        CONSTRAINT [FK_UserSignatureKeyPair_User] FOREIGN KEY ([UserId]) REFERENCES [dbo].[User] ([Id])
    );
END
GO
```

Reply:
@quexten Looks like most of these comments on the migration script aren't currently applied. 

Maybe got wiped out in the force push?

Reply:
I'm not entirely sure what happened here, but they have been re-applied.